### ISSUE TITLE
about the speed

### ISSUE THREAD

**Issue by speedcell4** [cheonbok94/Pytorch-A-Theoretically-Grounded-Application-of-Dropout-in-Recurrent-Neural-Networks#1]
have you tested the speed of your customized LSTM? 
since your implementation is similar to AllenNLP's this part and I have tried AllenNLP's implementation, I found it's very slow.
with `pytorch.nn.LSTM` my model finishes 1 epoch training in 40 seconds, but with `allennlp.modules.stacked_bidirectional_lstm.StackedBidirectionalLstm` it takes 8 min 20 seconds. 
how about your code?
